src/seedu/addressbook/AddressBook.java

```diff
@@ -26,30 +27,19 @@
  */
 
 /**
- * This class is used to maintain a list of person data which are saved
- * in a text file.
+ * This class is used to maintain a list of person data which are saved in a
+ * text file.
  **/
 public class AddressBook {
 
-    /**
-     * Default file path used if the user doesn't provide the file name.
-     */
-    private static final String DEFAULT_STORAGE_FILEPATH = "addressbook.txt";
-
-    /**
-     * Version info of the program.
-     */
-    private static final String VERSION = "AddessBook Level 1 - Version 1.0";
-
-    /**
-     * A decorative prefix added to the beginning of lines printed by AddressBook
-     */
-    public static final String LINE_PREFIX = "|| ";
+    private enum PersonProperty {
+        NAME, PHONE, EMAIL
+    }
 
     /**
      * A platform independent line separator.
      */
-    private static final String LS = System.lineSeparator() + LINE_PREFIX;
+    // private static final String LS" = System.lineSeparator() + "|| ";
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.